Sources/SwiftSyntax/Convenience.swift

```diff
@@ -68,6 +68,79 @@ extension EnumCaseParameterSyntax {
   }
 }
 
+extension FloatLiteralExprSyntax {
+  ///  A computed property representing the floating-point value parsed from the associated `literal.text` property.
+  ///
+  /// - Returns: A double value parsed from the associated`literal.text`, or `nil` if the text cannot be parsed as a double.
+  public var representedLiteralValue: Double? {
+    guard !hasError else { return nil }
+
+    let floatingDigitsWithoutUnderscores = literal.text.filter {
+      $0 != "_"
+    }
+
+    return Double(floatingDigitsWithoutUnderscores)
+  }
+}
+
+extension IntegerLiteralExprSyntax {
+  public enum Radix {
+    case binary
+    case octal
+    case decimal
+    case hex
+
+    public var size: Int {
+      switch self {
+      case .binary: return 2
+      case .octal: return 8
+      case .decimal: return 10
+      case .hex: return 16
+      }
+    }
+
+    fileprivate var offset: Int {
+      switch self {
+      case .binary, .hex, .octal:
+        return 2
+      case .decimal:
+        return 0
+      }
+    }
+  }
+
+  public var radix: Radix {
```

Comment:
Is this supposed to be `public`? If this is not necessary, I'd suggest not to expose this, or we'd need to update the release notes. Same for `enum Radix`.

Reply:
It was already public as part of `Sources/SwiftRefactor/IntegerLiteralUtilities.swift`?
So I'm not sure if it would be a breaking change to make it private

Reply:
Oooh, I see you just moved the code. No worries. Sorry for confusion.

Reply:
This would have needed importing `SwiftRefactor` before, so... maybe also worth release noting then?

Reply:
import `SwiftRefactor`?

Not sure I understand. 

It was moved from `SwiftRefactor` to `SwiftSyntax`.

Where it's used `IntegerLitralUtilities` is already `import SwiftSyntax`

Reply:
Yes, my point is that to use it previously you would have needed to import `SwiftRefactor`, that is no longer the case.

Reply:
Ah okay, but then the "maybe also worth release noting then?" do you mean adding a notice in the release note? 
Just so I understand. Sorry for confusing 😅

Reply:
Yes, Ben is suggesting to add an entry to the release notes.